Added note about auth token being one-time use (#39)
splitpayments/api-documentation: source/index.html.md

    Finally, the authorisation code can then be exchanged for an access token and refresh token pair by POSTing to: `https://go.sandbox.split.cash/oauth/token`

>     **NOTE** The authorisation code that you get will be a ONE-TIME use code. It will not work again if you try to POST it a second time.

Reviewer: Suggestion:

```suggestion
    **Note:** The authorisation code is a ONE-TIME use code. It will not work again if you try to POST it a second time.
```
